src/FlatSelect/components/SelectSearchList/types.ts

```diff
@@ -0,0 +1,24 @@
+// tslint:disable-next-line: interface-name
+export interface SelectSearchListProps {
+	onInputChange: (changedValue: string) => void;
+	className?: string;
+	searchTextValue: string;
+	shouldSearchTextBeSelected?: boolean;
+	onScroll?: (element: Element) => () => void;
+	selectionMode?: SelectionMode;
+	headerInfo?: string | JSX.Element;
+	clearFilterHandler?: React.MouseEventHandler<{}>;
```

Comment:
Нет смысла указывать объект в качестве типа тут, нужно передать сюда либо конкретный тип тэга, либо использовать значение по умолчанию, а это Element.
```suggestion
	clearFilterHandler?: React.MouseEventHandler;
```